setup.yaml

```diff
@@ -0,0 +1,6 @@
+before_script:
```

Comment:
I actually don't want this all to run as a before script, but instead to run only in the tasks that run node.

If this runs as a before script, I need all my ci tasks to have npm installed, even if they don't use it (like that node-flyway image which doesn't actually use node).